opencog/cython/opencog/type_constructors.pyx

```diff
@@ -0,0 +1,132 @@
+
+# DO NOT EDIT THIS FILE! This file was automatically generated from atom
+# definitions in types.script by the macro OPENCOG_ADD_ATOM_TYPES
```

Comment:
It doesn't really, but then it won't show up in the source on github and it will be far more mysterious where these types come from.

I contemplated adding it to .gitignore but decided against it.

Not a big deal but having a copy on github so people can search and find it via google, seemed like a good thing.